lang/python/python-mysql/Makefile

```diff
@@ -1,20 +1,20 @@
 #
-# Copyright (C) 2007-2014 OpenWrt.org
+# Copyright (C) 2007-2018 OpenWrt.org
 #
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
 
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=MySQL-python
-PKG_VERSION:=1.2.5
+PKG_NAME:=mysqlclient
```

Comment:
why not call this `python-mysql` ?
otherwise this could be confused with a official `mysqlclient`

Reply:
to add to this, i admit that you will need to change `PKG_SOURCE:=mysqlclient-$(PKG_VERSION).zip`;
but it's a bit better than the confusion this could create;